docker/documentation-site-builder/docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+if [ ! -f ../../docs/Gemfile ]; then
+  echo "ERROR: Gemfile not found"
+  exit 1
+fi
+
+bundle install --retry 5 --jobs 20
+
```

Comment:
No need for this installation after you refactor the Dockerfile to install gems at build time.

Reply:
Eliminated